app/views/shared/steps/_page_title.html.erb

```diff
@@ -0,0 +1,9 @@
+<% page_title = '' %>
+
+<% if links.length %>
```

Comment:
please don't do this, it will always evaluate to true, what you looking for is `if links.any?`

Reply:
Actually, in this case the `if` isn't necessary at all. I will remove it.
I have copied this from the existing `_step_breadcrumb.html.erb`. I can change it in there.